static/app/views/settings/project/projectKeys/details/loaderSettings.tsx

```diff
@@ -0,0 +1,266 @@
+import {Fragment, useCallback, useEffect, useState} from 'react';
+
+import {
+  addErrorMessage,
+  addLoadingMessage,
+  addSuccessMessage,
+} from 'sentry/actionCreators/indicator';
+import Access from 'sentry/components/acl/access';
+import FieldGroup from 'sentry/components/forms/fieldGroup';
+import FieldHelp from 'sentry/components/forms/fieldGroup/fieldHelp';
+import BooleanField from 'sentry/components/forms/fields/booleanField';
+import SelectField from 'sentry/components/forms/fields/selectField';
+import ExternalLink from 'sentry/components/links/externalLink';
+import TextCopyInput from 'sentry/components/textCopyInput';
+import {t, tct} from 'sentry/locale';
+import {space} from 'sentry/styles/space';
+import getDynamicText from 'sentry/utils/getDynamicText';
+import handleXhrErrorResponse from 'sentry/utils/handleXhrErrorResponse';
+import useApi from 'sentry/utils/useApi';
+import {ProjectKey} from 'sentry/views/settings/project/projectKeys/types';
+
+type Props = {
+  keyId: string;
+  organizationId: string;
+  projectId: string;
+  projectKey: ProjectKey;
+};
+
+export enum DynamicSDKLoaderOption {
+  HAS_DEBUG = 'hasDebug',
+  HAS_PERFORMANCE = 'hasPerformance',
+  HAS_REPLAY = 'hasReplay',
+}
+
+export const sdkLoaderOptions = {
+  [DynamicSDKLoaderOption.HAS_PERFORMANCE]: {
+    label: t('Enable Performance Monitoring'),
+    requiresV7: true,
+  },
+  [DynamicSDKLoaderOption.HAS_REPLAY]: {
+    label: t('Enable Session Replay'),
+    requiresV7: true,
+  },
+  [DynamicSDKLoaderOption.HAS_DEBUG]: {
+    label: t('Enable Debug Bundles & Logging'),
+    requiresV7: false,
+  },
+};
+
+export function LoaderSettings({keyId, organizationId, projectId, projectKey}: Props) {
```

Comment:
is `organizationId` really an id or actually `organizationSlug`? same for `projectId`

Reply:
I think they are actually slugs (this is often really confusing 😅 ) I'll double check!

Reply:
Updated this to be `orgSlug`!